lib/collection/src/collection/telemetry.rs

```diff
@@ -0,0 +1,41 @@
+use common::types::{DetailsLevel, TelemetryDetail};
```

Comment:
This small refactor is not necessary anymore, but I decided to keep it anyways since it declusters the mod.rs file.
If you prefer it to be reverted, let me know!